qtranslate.php

```diff
@@ -63,7 +63,7 @@
 }
 
 require_once QTRANSLATE_DIR . '/src/init.php';
-add_action( 'plugins_loaded', 'qtranxf_init_language', 2 ); // User is not authenticated yet, high priority needed.
+add_action( 'init', 'qtranxf_init_language', 2 );
```

Comment:
The consequence of that change is MUCH bigger than just delaying the text domain.

Reply:
This should do the trick in the init.php, please check it
```
add_action( 'init', 'qtranxf_load_plugin_textdomain' );
```

Reply:
Oh, I'm sorry, I have approved this review and didn't mentioned, that it is for "qtranslate.php" file, not for "init.php".

This fix:

`add_action( 'init', 'qtranxf_load_plugin_textdomain' );`

For "init.php" file works perfectly!

Reply:
Nothing to be sorry about! Thanks for testing :)